modules/nf-core/plink/genome/main.nf

```diff
@@ -0,0 +1,50 @@
+process PLINK_GENOME {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/plink:1.90b6.21--h031d066_5':
+        'biocontainers/plink:1.90b6.21--h031d066_5' }"
+
+    input:
+    tuple val(meta), path(bed), path(bim), path(fam)
+
+    output:
+    tuple val(meta), path("*.genome"), emit: genome
+    path "versions.yml"              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args   = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    plink \\
+        --bed ${bed} \\
+        --bim ${bim} \\
+        --fam ${fam} \\
+        $args \\
+        --genome \\
+        --threads $task.cpus \\
+        --out ${prefix}
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        plink: \$(echo \$(plink --version 2>&1) | sed 's/^PLINK v//' | sed 's/..-bit.*//' )
+    END_VERSIONS
+    """
+
+    stub:
+    def args   = task.ext.args ?: ''
```

Comment:
```suggestion
```
The language server will likely mention that these aren't being used. So think you can remove.

Reply:
$args are use in the command

Reply:
Does it being used in the command mean it is needed for the stub? This is all new to me.

Reply:
oh sorry this is the stub...so its not being used.... agree with @Joon-Klaps can be removed